docs/devGuide/settingUp.md

```diff
@@ -10,39 +10,58 @@
 
 <div class="lead">
 
-This page explains how to set up your development environment to start contributing to MarkBind.
+This page explains how to set up your development environment to start
+contributing to MarkBind.
+
 </div>
 
 ## Prerequisites
 
-1. **Node.js** ({{ node_version }} or higher) with<br>
-   **npm** v7 or higher
-1. **Java** 8 or later, and<br>
-   **Graphviz** v2.38 or later<br>
-   %%(The above two are required for one of the third-party library used by MarkBind)%%
-1. **Verify** that all tools are accessible and the versions are as expected by running these commands in the console.
-   * `node --version`
-   * `npm --version`
-   * `java --version`
-   * `dot -V` (for Graphviz)
+1. **Node.js** ({{ node_version }} or higher) with<br> **npm** v7 or higher
+1. **Java** 8 or later, and<br> **Graphviz** v2.38 or later<br> %%(The above two
+   are required for one of the third-party library used by MarkBind)%%
+1. **Verify** that all tools are accessible and the versions are as expected by
+   running these commands in the console.
+   - `node --version`
+   - `npm --version`
+   - `java --version`
+   - `dot -V` (for Graphviz)
```

Comment:
The stylistic choice of making the text after the line-break be in the same line as the previous text unfortunately made the raw text too crammed in (for instance, see the `%%()%%` part). We can let it be the original and make the text after the line-break be in its own line for easier development readability.

The changed in list notation from `*` to `-` is also an unnecessary change as they both work the same.